app/templates/summary.html

```diff
@@ -30,15 +30,15 @@ <h2 class="u-fs-l">{{ _("Please remember to submit these answers") }}.</h2>
 {% elif content.summary.summary_type in ('SectionSummary', 'AnswerSummary') %}
 <div>
     {% set group = content.summary.groups %}
-    <h1 class="u-fs-l">{{content.summary.title}}</h1>
+    <h1 class="u-fs-l question__title">{{content.summary.title}}</h1>
```

Comment:
@hamishtaplin is it okay to use this style on the summary page. Seems to fix the problem, but semantically this is a summary title not a question title? 🤷‍♂

Reply:
Not really. Survey Runner follows [The BEM convention](http://getbem.com/naming/) of writing CSS/components and so `question__title` should only be used within the scope of a `question` block.

Either we don't support it here, or we should make a CSS change.

Reply:
Now changed so its within scope of `question` block